snmp/tests/test_check.py

```diff
@@ -453,3 +457,20 @@ def test_cast_metrics(aggregator):
     aggregator.assert_metric('snmp.cpuload2', value=0.06)
 
     aggregator.all_metrics_asserted()
+
+
+def test_profile(aggregator):
+    instance = common.generate_instance_config([])
+    instance['profile'] = 'profile1'
+    with temp_dir() as tmp:
+        profile_file = os.path.join(tmp, 'profile1.yaml')
+        with open(profile_file, 'w') as f:
+            f.write(yaml.safe_dump(common.SUPPORTED_METRIC_TYPES))
```

Comment:
(minor) IMHO it would be helpful to have a fixture file, so we have a concret example of how a profile file look like.

Reply:
I think it'd be better handled by doc later on.